# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/make-config.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN vdr-1.3.5/Make.config vdr-1.3.5-work/Make.config
--- vdr-1.3.5/Make.config	1970-01-01 01:00:00.000000000 +0100
+++ vdr-1.3.5-work/Make.config	2004-03-04 17:55:39.418586642 +0100
@@ -0,0 +1,25 @@
+#
+# User defined Makefile options for the Video Disk Recorder
+#
+
+### The C compiler and options:
+
+CC       = gcc
+CFLAGS   = -O2
+
+CXX      = g++
+CXXFLAGS = -g -O2 -Wall -Woverloaded-virtual
+
+### The directory environment:
+
+DVBDIR   = /usr/include/linux/dvb
+MANDIR   = /usr/man
+BINDIR   = /usr/bin
+
+PLUGINDIR= ./PLUGINS
+PLUGINLIBDIR= $(PLUGINDIR)/lib
+VIDEODIR = /video
+
+REMOTE=LIRC
+#NO_KBD=1
+

#diff -ruN vdr-1.3.5/Makefile vdr-1.3.5-work/Makefile
#--- vdr-1.3.5/Makefile	2004-01-18 15:16:53.000000000 +0100
#+++ vdr-1.3.5-work/Makefile	2004-03-04 17:52:03.923622148 +0100
#@@ -29,7 +29,7 @@
# 
# -include Make.config
# 
#-INCLUDES = -I$(DVBDIR)/include
#+INCLUDES = -I/usr/src/linux/include -I$(DVBDIR)
# 
# SILIB    = $(LSIDIR)/libsi.a
# 
